### PR TITLE
Add Denodo connection

### PR DESCRIPTION
**Pull request by scruz-denodo**
Following work done at #11882

### CODE REVIEW

**Comment by serge-rider** [comment]
Thank you for the driver config update.
We will test it in 21.3.5.

File: plugins/org.jkiss.dbeaver.ext.denodo.ui/src/org/jkiss/dbeaver/ext/denodo/views/DenodoConnectionPage.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 /**
  * DenodoConnectionPage
  */
-public class DenodoConnectionPage extends ConnectionPageAbstract implements IDialogPageProvider, IDataSourceConnectionTester {
+public class    DenodoConnectionPage extends ConnectionPageAbstract implements IDialogPageProvider, IDataSourceConnectionTester {
```
Comment (flambchap) [inline plugins/org.jkiss.dbeaver.ext.denodo.ui/src/org/jkiss/dbeaver/ext/denodo/views/DenodoConnectionPage.java]: I'm not sure that we need this change.

Comment (scruz-denodo) [reply plugins/org.jkiss.dbeaver.ext.denodo.ui/src/org/jkiss/dbeaver/ext/denodo/views/DenodoConnectionPage.java]: Sure, I will change it.

Comment (scruz-denodo) [reply plugins/org.jkiss.dbeaver.ext.denodo.ui/src/org/jkiss/dbeaver/ext/denodo/views/DenodoConnectionPage.java]: Done f0a59174e71abf23909d75fab3f59390e341f7b7

**Pull request merged**